tads2/output.c

```diff
@@ -2470,7 +2470,8 @@ static int outformatlen_stream(out_stream_info *stream,
                 else if (!stricmp(tagbuf, "b")
                          || !stricmp(tagbuf, "i")
                          || !stricmp(tagbuf, "em")
-                         || !stricmp(tagbuf, "strong"))
+                         || !stricmp(tagbuf, "strong")
+                         || !stricmp(tagbuf, "tt"))
```

Comment:
Would supporting `<pre>` be as simple as `<tt>`? I assume supporting `<font>` would be more complicated, but it would be great if we can do that too.

Reply:
The built-in HTML parser already recognises `<pre>` for the purpose of inserting paragraph breaks. I think I have an idea of how to shove the monospace attribute in, for T3 at least, but I need to figure out how to build Gargoyle again so I can test it.

Reply:
Yup, this works.